front-end/src/app/layout/layout.component.html

```diff
@@ -1,39 +1,39 @@
 <app-banner></app-banner>
-<div [ngClass]="isVisible ? '': 'container'">
-    <div class="row">
-        <div class="col-xxl-2 col-xl-3 col-lg-4 col-md-5 col-sm-5  fec-background-gradient" *ngIf="isVisible">
-            <app-sidebar></app-sidebar>
-        </div>
-        <div [ngClass]="isVisible ? 'col-xxl-10 col-xl-9 col-lg-8 col-md-7 col-sm-7  ' : 'col-12'">
-            <div class="col-12 header-container">
-                <app-header></app-header>
+<div [ngClass]="showSidebar ? '' : 'container'">
```

Comment:
is this class condition necessary? seems to work with just <div>

Reply:
@sasha-dresden Is there a accessibility or other reason for the dynamic 'container' class?

Reply:
@toddlees I spoke to Sasha and that ngClass is necessary to manage the responsive design properly as the screen is expanded or shrunk.

Reply:
So, two things on this. 
For the dynamic container class. That's there to prevent content stretching on large screen sizes when there's no side banner. 
Two, Github really made this change confusing. Looking at what actually changed, it looks like the only actual change was all of the calls to "isVisible" were changed to "showSidebar".